File: 截屏通知:测试/截屏通知:测试/ViewController.m
//
//  ViewController.m
//  截屏通知:测试
//
//  Created by coco on 16/5/27.
//  Copyright © 2016年 XHJ. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *screen;

@end

@implementation ViewController
static int i = 1;

- (void)viewDidLoad {
    [super viewDidLoad];
    //UIApplicationUserDidTakeScreenshotNotification
    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(takeScreenshot:) name:UIApplicationUserDidTakeScreenshotNotification object:nil];
    // Do any additional setup after loading the view, typically from a nib.
}
- (void)takeScreenshot:(NSNotification *)notification {
    self.screen.text = [NSString stringWithFormat:@"%d", i++];
}
- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: dictionaryToModel_Swift3.0/dictionaryToModel_Swift3.0/ViewController.swift
//
//  ViewController.swift
//  dictionaryToModel_Swift3.0
//
//  Created by coco on 16/10/10.
//  Copyright © 2016年 XHJ. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
//        let dic = ["name": "xhj", "age": "20", "sex": "男", "family": ["mother": "abc", "father": "def"], "hot": [["name": "你好"], ["name": "我也好"]]] as [String : Any]
        let dic = ["name": "xhj", "age": "20", "sex": "男", "family": ["mother": "abc", "father": "def"]] as [String : Any]
        let model = User.dictionaryToModel(dic: dic) as! User
        
        print(model.keyValues)

        let aaa = ["1": "a", "2": "b", "3": "c", "4": ["d", "e", "f"]] as [String : Any]
        print(aaa.keyValues)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

